package.json

```diff
@@ -44,5 +46,10 @@
     "react-router": "^2.4.0",
     "sinon": "^1.17.2",
     "sinon-chai": "^2.8.0"
+  },
+  "publishr": {
+    "dependencies": [
+      "^builder"
+    ]
```

Comment:
Ok. Thanks for refresher. I think we can skip for now. When we tackle "provide es6 sources" we can figure out what we want, but having raw `src` (needing babel + webpack) is fine for now in published module.